apps/system/style/pin_page_system_dialog.css

```diff
@@ -0,0 +1,68 @@
+#pin-page-dialog section {
+  position: absolute;
+  top: 0;
+  left: 0;
+  display: block;
+  overflow: hidden;
+  width: 100%;
+  height: 100%;
+  text-align: center;
+  background-color: #f4f4f4;
+}
+
+#pin-page-dialog section .container > .url {
+  font-size: 1.4rem;
+  font-weight: normal;
+  font-style: italic;
+  color: #5f5f5f;
+  text-overflow: ellipsis;
+  width: calc(100% - 8rem);
+  margin: 0 auto;
+  white-space: nowrap;
+  overflow: hidden;
+  padding: 0;
+}
+
+#pin-card-container {
+  display: block;
+  overflow: hidden;
+  width: 100%;
+  margin: 2rem 0 3rem;
+  padding-bottom: .5rem;
+  padding-top: 2rem;
+}
+
+gaia-pin-card {
+  display: block;
+  width: 14rem;
+  height: 10.5rem;
+  margin: 0 auto;
+}
+
+button[data-action="pin"] {
+  position: relative;
+  margin: 3rem auto;
+  width: -moz-fit-content;
+  background-color: white;
+  display: block;
+  float: none;
+  font-size: 1.6rem;
+  font-style: italic;
+
+}
+
+button[data-action="pin"] {
+  -moz-padding-start: 4rem;
+  -moz-padding-end: 2rem;
```

Comment:
Not that I'm aware off. We are using this all over the place for RTL languages support.